lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -65,7 +65,7 @@ function createGroup(group, callback) {
             'timezone',
             'commands',
             'lazy',
-            'active',
+            'attributes',
```

Comment:
Should the `id` property be `_id` instead (since I think you decided to use the standard `_id` property)? :)